Pipfile

```
@@ -9,12 +9,13 @@ Flask-RESTful = "==0.3.9"
 apispec = {extras = ["yaml"], version = "==5.0"}
 apispec-webframeworks = "==0.5.2"
 bcrypt = "==3.2.2"
-flask = "==2.1"
+flask = "==2.2.2"
 flask-cors = "==3.0.10"
 flask-marshmallow = "==0.14"
 flask-socketio = "==5.1.1"
```

```suggestion
flask-socketio = "==5.3.0"
gevent = "==21.12.0"
gevent-websocket = "==0.10.1"
```
This should be ok, I suppose.
Does it necessary to move `gevent` and `gevent-websocket` below `flask-talisman` line?